types/src/get_with_proof.rs

```diff
@@ -329,9 +329,13 @@ fn verify_get_txn_by_seq_num_resp(
             )
         },
         (None, Some(proof_of_current_sequence_number)) => {
+                let account_resource: AccountResource =
+                TryFrom::<&AccountStateBlob>::try_from(proof_of_current_sequence_number.blob.as_ref().ok_or_else(|| format_err!("Missing proof_of_current_sequence_number.blob"))?)?;
```

Comment:
We shouldn't return error here if `proof_of_current_sequence_number` is `None`. That means the account given by the request does not exist, which is valid.

Reply:
What should we do in this case that `proof_of_current_sequence_number` is `Some`, but `proof_of_current_sequence_number.blob` is `None`--just return early with `Ok())`? The old code would have given you `let sequence_number_in_ledger = 0` here, but not sure if that makes sense.

Reply:
Makes sense, thanks for the detailed explanation!